web/src/components/Sidebar.js

```diff
@@ -10,7 +10,7 @@ import { uiColors } from "@leafygreen-ui/palette";
 
 export default function Sidebar({ currentProject, setCurrentProject, setIsEditingPermissions }) {
   const projects = useProjects();
-  const { app, refresh } = useRealmApp();
+  const app = useRealmApp();
```

Comment:
```suggestion
  const { currentUser, logOut } = useRealmApp().app;
```
would probably clean things up + is more inline w/ es6 syntax

Reply:
This would work but I actually want everything contained within the `app` object so that we build familiarity with the pattern we use in other docs examples.